app/views/participants/validations/exempt_from_induction.html.erb

```diff
@@ -0,0 +1,36 @@
+<% title = "You do not need to serve statutory induction for early career teachers" %>
+<% content_for :title, title %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <div class="govuk-panel govuk-panel--confirmation">
+      <h1 class="govuk-panel__title">
+        You do not need to serve statutory induction for early career teachers
+      </h1>
+    </div>
+
+    <p class="govuk-body">
+      Your Teaching Regulation Agency record shows that you do not need to complete a statutory induction, or take part in ECF-based training.
+    </p>
+
+    <p>
+      This is likely to be because of the way in which DfE recognised your qualified
+      teacher status (QTS).
+    </p>
+
+    <p>
+      Annex A of the Induction for early career teachers guidance explains who’s
+      exempt:  https://www.gov.uk/government/publications/induction-for-early-career-
+      teachers-england
```

Comment:
```suggestion
      exempt:  <%= govuk_link_to "https://www.gov.uk/government/publications/induction-for-early-career-
      teachers-england" %>
```

Should this be a link?

Reply:
It wasn't in the content I was given, I'll check.